specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -5529,6 +5526,51 @@
         }
       }
     },
+    "InvoiceSectionListWithCreateSubPermissionResult": {
+      "type": "object",
+      "description": "Result of listing invoice section properties with create subscription permission.",
+      "properties": {
+        "value": {
+          "description": "The list of invoice section properties with create subscription permission.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InvoiceSectionWithCreateSubPermission"
+          }
```

Comment:
missing:
        "nextLink": {
          "description": "The link (url) to the next page of results.",
          "type": "string",
          "readOnly": true
        }

Reply:
Sure, Added.